src/Validator/Rules/UniqueDirectivesPerLocation.php

```diff
@@ -26,16 +34,41 @@ public function getSDLVisitor(SDLValidationContext $context)
 
     public function getASTVisitor(ASTValidationContext $context)
     {
+        $uniqueDirectiveMap = [];
+
+        $schema            = $context->getSchema();
+        $definedDirectives = $schema !== null
```

Comment:
Personally, I'd prefer not to use negation in conditions where eligible. Reduces cognitive load while reading it.

Reply:
Generally agree, although in this case `$schema` is usually non-null. It makes sense to have the happy path first and the fallback afterwards.

Reply:
I will agree with whomever merges some of my PRs first, so I don't have to generate my basefiles again...